### PR TITLE
Add Makefile for convenience

### PR DESCRIPTION
**Pull request by okurz**
New developers would have a hard time finding necessary dependencies and
commands to execute by needing to look into GHA definitions so instead
let's provide a convenient Makefile with help text.

### CODE REVIEW

**Review by Martchus — DISMISSED** [review]

**Comment by Martchus** [comment]
Although I must say that I find putting `sudo` and distribution-specific and package-manager-specific commands into the Makefile problematic. I would have at least prefixed the targets with e.g. `-debian` and `-cpanm`.

**Comment by okurz** [comment]
> Although I must say that I find putting `sudo` and distribution-specific and package-manager-specific commands into the Makefile problematic. I would have at least prefixed the targets with e.g. `-debian` and `-cpanm`.

err, do you mean prefix or suffix? Should I rename `install-deps-perl` to `install-deps-cpanm` or `install-deps-perl-cpanm` or `cpanm-install-deps-perl`?

And what about sudo? Keep it, just add suffix/prefix debian?

**Comment by Martchus** [comment]
Sorry, I mean suffix (although it doesn't really matter). And I suppose `sudo` just falls under `debian` :-)

**Review by Martchus — DISMISSED** [review]

**Comment by okurz** [comment]
Fixed missing name adaption in github action workflow. Sorry, you need to re-review as this project is configured to remove all approvals on pushed changes.

**Review by baierjan — CHANGES_REQUESTED** [review]

**Comment by okurz** [comment]
@baierjan incorporated both of your suggestions

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -31,31 +31,17 @@ jobs:
...
       - name: perl -V
         run: perl -V
       - name: Install dependencies
         run: |
-          sudo apt-get update
-          sudo apt-get install -y libmagic-dev ruby-sass
-          cpanm -n --installdeps .
-          cpanm -n Test::Deep
-          cpanm -n Devel::Cover::Report::Coveralls
+          make install-deps-debian install-deps-cpanm
```
Comment (baierjan) [inline .github/workflows/linux.yml:40]: 🤔 

```suggestion
          make install-deps-ubuntu install-deps-cpanm
```

**Review by baierjan — APPROVED** [review]

**Pull request merged**